src/Wpf.Ui/Controls/IconElements/SymbolIcon.cs

```diff
@@ -0,0 +1,87 @@
+﻿// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+// Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+// All Rights Reserved.
+
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows;
+using Wpf.Ui.Common;
+using Wpf.Ui.Extensions;
+
+namespace Wpf.Ui.Controls.IconElements;
+
+/// <summary>
+/// Represents a text element containing an icon glyph.
+/// </summary>
+[ToolboxItem(true)]
+[ToolboxBitmap(typeof(SymbolIcon), "SymbolIcon.bmp")]
+public sealed class SymbolIcon : FontIcon
```

Comment:
Someone, someday, somehow, might want to alter this class. I believe that there should be no sealed classes in an open-source project.  We should open all such classes.